hadoop-mr-s3/src/main/java/org/apache/hadoop/mapred/DirectFileOutputCommitter.java

```diff
@@ -0,0 +1,88 @@
+package org.apache.hadoop.mapred;
```

Comment:
Can we move this outside `org.apache.hadoop.mapred`? It really helps while debugging (esp. on cases like Qubole where they might have their own version and we don't know which class implementation got picked). Sorry, if I sound  little paranoid.